src/wui/story_message_box.cc

```diff
@@ -72,16 +72,27 @@ StoryMessageBox::StoryMessageBox(Widelands::Game* game,
 	}
 	move_inside_parent();
 	textarea_.focus();
+
+	if (!is_modal()) {
+		resume_game();
+	}
```

Comment:
Because the constructor does this:

```
	if (coords != Widelands::Coords::null()) {
		game_->get_ipl()->map_view()->scroll_to_field(coords, MapView::Transition::Jump);
	}
```

I didn't want player actions to interfere here, might result in unexpected behavior.